packages/commercetools/composables/src/useUserAddress/index.ts

```diff
@@ -0,0 +1,9 @@
+import { UseUserAddress } from '@vue-storefront/interfaces';
+import { Address } from '../types/GraphQL';
+import { useUserAddressFactory } from '@vue-storefront/factories';
+import { Customer } from '@vue-storefront/commercetools-api/lib//types/GraphQL';
+import { params } from './factoryParams';
+
+const useUserAddress: () => UseUserAddress<Address> = useUserAddressFactory<Customer, Address>(params);
+
+export default useUserAddress;
```

Comment:
Why not just like this?

```suggestion
export default useUserAddressFactory<Customer, Address>(params);
```